### ISSUE TITLE
Numeric value: "Read information about a meter in reading mode" (Meter, Test 5)

### ISSUE THREAD

**Issue by AlyssaGourley** [w3c/aria-at#806]
## Description of Behavior

For the numeric value, NVDA reports "36.1665" but the criteria states it should report the value as "36.166519936288." I failed this because the value repoorted with NVDA was not exact.

## Test Setup

- Test File: [test-05-read-information-about-a-meter-reading-nvda.collected.html](https://aria-at.netlify.app/tests/meter/test-05-read-information-about-a-meter-reading-nvda.collected.html)
- AT: NVDA (version 2022.2)
- Browser: Chrome (version 104.0.0)

## Review Conflicts for "Read information about a meter in reading mode"

1.  ### Assertion Results for "Insert+Tab" Command and "Numeric value '36.166519936288' is conveyed" Assertion
    
    *   Tester AlyssaGourley recorded output "Central Processing Unit (CPU) Usage progress bar 36.1665 focused" and marked assertion as failing with incorrect output.
    *   Tester IsaDC recorded output "Central Processing Unit (CPU) Usage progress bar 36.1665 focused " and marked assertion as passing.
2.  ### Assertion Results for "Insert+Up" Command and "Numeric value '36.166519936288' is conveyed" Assertion
    
    *   Tester AlyssaGourley recorded output "Progress Bar 36.1665" and marked assertion as failing with incorrect output.
    *   Tester IsaDC recorded output "progress bar 36.1665 " and marked assertion as passing.

**Comment by jscholes** [comment]
@AlyssaGourley Thanks for flagging this!  It was an incorrect result entry on our end that has now been changed, so I'll go ahead and close this issue.
